python/example/single_cell_stdp.py

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env python3
```

Comment:
Can we add the example to github actions? https://github.com/arbor-sim/arbor/blob/master/.github/workflows/basic.yml

Reply:
done